packages/@datapull/map-transformer/index.js

```diff
@@ -14,7 +15,11 @@ class MapTransformer {
           r.push(transformedItem);
         });
       });
+      return Promise.resolve(r);
+    }
 
+    if (this.config === 'mergeArrays') {
+      r = [].concat.apply([], data);
```

Comment:
Isn't this identical to `[].concat(...data)`? Also, why assign to `r` if you're immediately returning it in the next line?

Reply:
fixed